ktor-server/ktor-server-plugins/ktor-server-auth-ldap/jvm/test/io/ktor/tests/auth/ldap/LdapAuthTest.kt

```diff
@@ -27,148 +28,144 @@ import kotlin.test.*
 class LdapAuthTest {
```

Comment:
This test fails for me both before refactoring and after it, so I cannot check the correctness of the refactoring.